docs/pages/database-access/guides/mysql-cloudsql.mdx

```diff
@@ -302,3 +355,21 @@ $ tsh db logout cloudsql
 $ tsh db logout
 ```
 
+## Troubleshooting
+
+### Error when connecting to a replica instance
+
+You may encounter the following error when connecting to a replica instance:
+
+```code
+$ tsh db connect --db-user root -n test cloudsql-replica 
+ERROR 1105 (HY000): Could not update Cloud SQL user "<username>" password:
+
+  The requested operation is not valid for a replica instance.
+
+...
+```
+
+Connecting as built-in database users with passwords are not supported for
```

Comment:
Similar to above, I think we just shouldn't say it's supported at all. Only keep it for backwards-compatibility reasons but out of user docs.